anime_downloader/commands/dl.py

```diff
@@ -42,7 +44,7 @@
     '--force-download', '-f', is_flag=True,
     help='Force downloads even if file exists')
 @click.option(
-    '--file-format', '-ff', default='{anime_title}/{anime_title}_{ep_no}',
+    '--file-format', '-ff', default='{animeinfo_anime_title}/{animeinfo_anime_title}_{provider}_{ep_no}',
```

Comment:
I dont think it's a good idea to change the default file format.

What would happen if the anime returned by MAL is the wrong one?

Reply:
Merely feature showcase, will fix before merge